api/src/main/java/ai/djl/modality/cv/translator/SemanticSegmentationTranslator.java

```diff
@@ -13,32 +13,27 @@
 package ai.djl.modality.cv.translator;
 
 import ai.djl.modality.cv.Image;
-import ai.djl.modality.cv.ImageFactory;
-import ai.djl.modality.cv.output.DetectedObjects;
+import ai.djl.modality.cv.output.Segmentation;
 import ai.djl.modality.cv.util.NDImageUtils;
 import ai.djl.ndarray.NDArray;
 import ai.djl.ndarray.NDList;
-import ai.djl.ndarray.NDManager;
-import ai.djl.ndarray.types.DataType;
 import ai.djl.ndarray.types.Shape;
 import ai.djl.translate.ArgumentsUtil;
 import ai.djl.translate.Transform;
+import ai.djl.translate.Translator;
 import ai.djl.translate.TranslatorContext;
-import ai.djl.util.RandomUtils;
 
-import java.nio.ByteBuffer;
 import java.util.Map;
 
 /**
- * A {@link BaseImageTranslator} that post-process the {@link NDArray} into {@link DetectedObjects}
- * with boundaries at the detailed pixel level.
+ * A {@link Translator} that post-process the {@link Image} into {@link Segmentation} with output
+ * mask representing the class that each pixel in the original image belong to.
  */
-public class SemanticSegmentationTranslator extends BaseImageTranslator<Image> {
+public class SemanticSegmentationTranslator extends BaseImageTranslator<Segmentation> {
 
     private final int shortEdge;
     private final int maxEdge;
 
-    private static final int CHANNEL = 3;
     private static final int CLASSNUM = 21;
```

Comment:
This should comes from model, for deeplabv3, the classes can be found: https://github.com/pytorch/vision/blob/main/torchvision/models/_meta.py#L1128-L1150